api/types/types.go

```diff
@@ -263,6 +263,8 @@ type ContainerJSONBase struct {
 	ExecIDs         []string
 	HostConfig      *runconfig.HostConfig
 	GraphDriver     GraphDriverData
+	SizeRw          string
```

Comment:
@calavera  If the container is actually empty, the size fields will not display even if user set the flag --size, is that ok?